### DISCUSSION TITLE
v0.10.5

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#933] [Releases]
**Full Changelog**: https://github.com/AsamK/signal-cli/compare/v0.10.4.2...v0.10.5

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.10.5'>v0.10.5</a>.</em>

**Comment by m-ueberall** [comment]
FTR, at least the Linux archive (`signal-cli-0.10.5-Linux.tar.gz`) has a **bad GPG signature**, see [issue #934](https://github.com/AsamK/signal-cli/issues/934).

**Reply by AsamK** [reply]
thanks, should be fixed now
